.travis.yml

```diff
@@ -0,0 +1,71 @@
+# This config file for Travis CI
+sudo: required
+
+services:
+  - docker
+
+git:
+  quiet: true
+  submodules: false
+
+notifications:
+  email:
+    recipients:
+      - levi.armstrong@swri.org
+
+env:
+  global:
+    - CCACHE_DIR=$HOME/.ccache
+    - ROS_REPO=ros
+    - NOT_TEST_INSTALL=true
+    - ADDITIONAL_DEBS="autotools-dev automake libpcre3-dev bison"
+
+jobs:
+  include:
+    - os: linux
+      dist: bionic
+      language: cpp
+      env:
+      - ROS_DISTRO="melodic"
+      - BADGE=xenial
```

Comment:
Does this matter?

Reply:
The BADGE tag will be used for creating badges for individual CI buids.